tests/config/hsc-gaia-config.py

```diff
@@ -1,8 +0,0 @@
-config.astrometryRefObjLoader.ref_dataset_name = "gaia"
-config.astrometryRefObjLoader.filterMap = {'u': 'phot_g_mean_mag',
-    'g': 'phot_g_mean_mag',
-    'r': 'phot_g_mean_mag',
-    'i': 'phot_g_mean_mag',
-    'z': 'phot_g_mean_mag',
-    'y': 'phot_g_mean_mag',
-}
```

Comment:
I assume this was not used?

Reply:
Yes, I should have removed this file on a previous ticket, but forgot to.